Community/alumni.md

```diff
@@ -0,0 +1,357 @@
+
+
+# CHAOSS Mentorship Alumni
+
+The CHAOSS Community thanks all mentees and mentors who participated in Google Summer of Code and Outreachy.
+
+
+## CHAOSS GSoC Students in 2020
+
+|Student|Project|Mentors|

```

Comment:
It would be nice to add a column `Year` :)

I think having the latest (2020) at the top would be most useful so that older projects move to the bottom.

```suggestion
|Student|Year|Project|Mentors|
```

Reply:
@GeorgLink I thought even the same but you mentioned it should be in chronological order. So according to timeline earliest year will be first and latest year will be last

Reply:
Reverse chronological order may have been more accurate then. Sorry 🤭